packages/atlas/src/components/WelcomeDialog/WelcomeDialog.tsx

```diff
@@ -0,0 +1,44 @@
+import { AppKV } from '@/components/AppKV/AppKV'
+import { Button } from '@/components/_buttons/Button'
+import { atlasConfig } from '@/config'
+import { absoluteRoutes } from '@/config/routes'
+import { usePersonalDataStore } from '@/providers/personalData'
+
+import { Divider, StyledAnchor, StyledDialogModal, StyledText, Wrapper } from './WelcomeDialog.styles'
+
+export const WelcomeDialog = () => {
+  const { isCookiesPopoverVisible, setCookiesAccepted } = usePersonalDataStore((state) => ({
+    isCookiesPopoverVisible: state.actions.getIsCookiesPopoverVisible(),
+    setCookiesAccepted: state.actions.setCookiesAccepted,
+  }))
+  return (
+    <StyledDialogModal
+      show={isCookiesPopoverVisible && !!atlasConfig.general.appContentFocus}
+      contentClassName="welcome-content"
+      size="small"
+    >
+      <Wrapper>
+        <AppKV />
+        <StyledText variant="h500" as="p" color="colorTextStrong">
+          Welcome to {atlasConfig.general.appName}
+        </StyledText>
+        <StyledText variant="t200" as="p" color="colorCoreNeutral200">
+          {atlasConfig.general.appName} is a {atlasConfig.general.appContentFocus} content-focused video platform that's
```

Comment:
according to the design, `appContentFocus` should have different color 
![image](https://user-images.githubusercontent.com/51168865/211804782-ca30ad1a-e3da-43e0-91df-0eb9250ecbc1.png)